operator/helper/severity_builder.go

```diff
@@ -23,13 +23,11 @@ func getBuiltinMapping(name string) severityMap {
 			"trace":       entry.Trace,
 			"debug":       entry.Debug,
 			"info":        entry.Info,
-			"notice":      entry.Notice,
-			"warning":     entry.Warning,
+			"notice":      entry.Info4,
```

Comment:
It's important that this map contains exactly the severity levels we support.

Remove `notice` and `catastrophe`. Add all the 2,3,4 levels.